app/CMakeLists.txt

```diff
@@ -374,3 +456,26 @@ cpp_pack_public_headers()
 if (NOT ANDROID AND NOT IOS)
   cpp_pack_library(flatbuffers "deps/app/external")
 endif()
+
+if (IOS)
+  set_target_properties(firebase_app PROPERTIES
+    FRAMEWORK TRUE
+  )
+
+  # Generate Framewrok Headers
```

Comment:
nit: Framework

There is quite a bit of post processing happening on the file names. Will be awesome to have a comment explaining the transformations.